src/gui/instrument/InstrumentTrackWindow.cpp

```diff
@@ -285,24 +285,30 @@ InstrumentTrackWindow::InstrumentTrackWindow( InstrumentTrackView * _itv ) :
 
 	QMdiSubWindow* subWin = getGUI()->mainWindow()->addWindowedWidget( this );
 	Qt::WindowFlags flags = subWin->windowFlags();
-	if (!m_instrumentView->isResizable()) {
-		flags |= Qt::MSWindowsFixedSizeDialogHint;
-		// any better way than this?
-	} else {
-		subWin->setMaximumSize(m_instrumentView->maximumHeight() + 12, m_instrumentView->maximumWidth() + 208);
-		subWin->setMinimumSize( m_instrumentView->minimumWidth() + 12, m_instrumentView->minimumHeight() + 208);
+
+	if (m_instrumentView->isResizable())
+	{
+		// TODO As of writing SlicerT is the only resizable instrument. Is this code specific to SlicerT?
+		const auto extraSpace = QSize(12, 208);
+		subWin->setMaximumSize(m_instrumentView->maximumSize() + extraSpace);
+		subWin->setMinimumSize(m_instrumentView->minimumSize() + extraSpace);
+
+		flags |= Qt::WindowMaximizeButtonHint;
```

Comment:
I assume, for symmetry reasons to the `else` case, you could clear the `Qt::MSWindowsFixedSizeDialogHint` here, too?

Reply:
Done with commit e85699f7872.